packages/circuits/package.json

```diff
@@ -23,6 +23,8 @@
     "bump-to": "yarn version --no-commit-hooks --no-git-tag-version --new-version"
   },
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.507.0",
```

Comment:
Not very important, but things like these could be in devDependency

Reply:
Yes, you're right!
I thought it was the case, I'll modify it